### PR TITLE
Add sponsoring page

### PR DESCRIPTION
**Pull request by lowsky**
implements #1096

### CODE REVIEW

**Review by Narigo — APPROVED** [review]
Looks good to me! 🎉 Thanks @lowsky !

File: src/components/explain-sponsoring/explain-sponsoring.module.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        list-style: disc;
+    }
+}
+.text_left {
+    text-align: left;
+}
+
+.sub_headline {
+  margin: 26px 0;
+}
```
Comment (Narigo) [inline src/components/explain-sponsoring/explain-sponsoring.module.scss:15]: Prettier / linter not complaining about the spaces? 😅

Comment (lowsky) [reply src/components/explain-sponsoring/explain-sponsoring.module.scss]: not complaining. - but reminds me that formatting was not enabled.

I guess, we should once format all files properly, with the latest prettier config we have.

Comment (lowsky) [reply src/components/explain-sponsoring/explain-sponsoring.module.scss]: I could do that ideally in a follow-up...

Comment (lowsky) [reply src/components/explain-sponsoring/explain-sponsoring.module.scss]: I guess this is now okay, and let's ship this page!

**Pull request merged**